components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```
@@ -23,6 +25,96 @@
+
+def _update_config_file_telemetry(config_file_path: pathlib.Path, disable: bool) -> None:
+    """
+    Updates the telemetry.disable setting in the config file.
+
+    :param config_file_path: Path to the config file.
+    :param disable: Whether to disable telemetry.
+    """
+    config_data = {}
+
+    if config_file_path.exists():
+        with open(config_file_path, "r") as f:
```

Ruff (0.15.12)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)